### DISCUSSION TITLE
What did you make your intro with

### DISCUSSION THREAD

**Discussion by markverb1** [ZI-Youtuber/ZI-Youtuber#2] [Q&A] [answered]
How did you make [this intro](https://m.youtube.com/watch?v=GVOExLlGTVk) ?

**Comment by Shluffy** [comment]
probably photoshop or another animating thing. @ZI-Youtuber how did you make it?

**Reply by markverb1** [reply]
Wait a minute ive seen this before. Its some free editing software. Idk the name of it tho

**Comment by zoraezimran** [comment, marked as answer]
It's called Windows Video Editor

**Reply by markverb1** [reply]
Oh the default video editor...

**Reply by zoraezimran** [reply]
@markverb1 yep
